Library/src/Common/String.cpp

```diff
@@ -163,6 +163,28 @@ List<String> String::Split(char Separator) const
 	return ImplPtr->Split(Separator);
 }
 
+List<String> String::Split(const char* Delimiters) const
+{
+	std::stringstream stringStream(ImplPtr->Text);
+	List<String> StringList;
+	std::string InputLine;
+
+	while (std::getline(stringStream, InputLine))
+	{
+		std::size_t Prev = 0, Pos;
+		while ((Pos = InputLine.find_first_of(Delimiters, Prev)) != std::string::npos)
+		{
+			if (Pos > Prev)
```

Comment:
Coding convention dictates we should always surround if blocks with curly brackets, even if they are single-line statements.